### PR TITLE
oAuthToken - initial commit

### PR DESCRIPTION
**Pull request by manioc2015**

### CODE REVIEW

File: components/schemas.yaml [hunk 1]
```diff
@@ -1069,6 +1029,28 @@ OrgAuthenticateRequest:
...
+          - code
+        type: object
+        properties:
+          grant_type:
+            type: string
+            enum:
+              - authorization_code
+          redirect_uri:
+            type: string
+            description: The client application request's callback url
```
Comment (kulyim) [inline components/schemas.yaml]: Grant type has to have a value of `auth_code`; we aren't supporting anything else currently.

File: components/schemas.yaml [hunk 2]
```diff
@@ -1069,6 +1029,28 @@ OrgAuthenticateRequest:
             type: string
             example: O0000c1x4UMGPvkY
 
+OAuthTokenRequest:
```
Comment (kulyim) [inline components/schemas.yaml]: The client will/SHOULD base64 encode its clientID and clientSecret into the authorization header as part of the request payload. Will we represent that somehow? Something still feels wrong about reproducing the oauth spec here.

Comment (manioc2015) [reply components/schemas.yaml]: the header is in oAuthToken.yaml scroll down

Comment (kulyim) [reply components/schemas.yaml]: lol, thanks

**Pull request merged**